qutebrowser/config/config.py

```diff
@@ -363,6 +363,8 @@ class ConfigManager(QObject):
             _get_value_transformer({'false': 'none', 'true': 'debug'}),
         ('ui', 'keyhint-blacklist'):
             _get_value_transformer({'false': '*', 'true': ''}),
+        ('hints', 'auto-follow'):
+            _get_value_transformer({'false': 'never', 'true': 'unique-match'}),
```

Comment:
I think `full-match` would be closer to the current `true` behaviour, right? On the other hand, probably `unique-match` makes more sense as the new default.